cmd/snap-confine/snap-confine.rst

```diff
@@ -129,7 +129,7 @@ FILES
 
 	Description of the mount profile.
 
-`/var/lib/snapd/seccomp/profiles/*`:
+`/var/lib/snapd/seccomp/bpf/*`:
```

Comment:
We need to list both `/var/lib/snapd/seccomp/profiles/*` *and* `/var/lib/snapd/seccomp/bpf/*` as well as the `snap-seccomp` tool. `/var/lib/snapd/seccomp/profiles/*` is all users will every look at and `/var/lib/snapd/seccomp/bpf/*` is all snap-confine will now ever look at. A quick note that the files in `/var/lib/snapd/seccomp/bpf/*` are generated by snapd and that modifications to `/var/lib/snapd/seccomp/profiles/*` need to then be compiled to `/var/lib/snapd/seccomp/bpf/*`.